docs/pip_sphinxext.py

```diff
@@ -108,9 +109,40 @@ def process_options(self):
         )
 
 
+class PipReqFileOptionsReference(PipOptions):
+
+    def process_options(self):
+        for option in SUPPORTED_OPTIONS:
+            if getattr(option, 'deprecated', False):
+                continue
+
+            opt = option()
+            opt_name = opt._long_opts[0]
+            if opt._short_opts:
+                short_opt_name = '{}, '.format(opt._short_opts[0])
+            else:
+                short_opt_name = ''
+
+            from_install = (
+                'install_'
+                if option not in cmdoptions.general_group['options'] else
```

Comment:
this is not ideal, but I am not sure how to figure out where to find the reference from :(

Reply:
I'll try to make it better today